app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    mask_needs_count = 0
+    for friend in friends:
+        try:
+            Cafe.visit_cafe(cafe, visitor=friend)
+
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
It can be better if you rewrite it use __str__ method in class VaccineError

Reply:
i will do it next time, thnx